api/CESMII.ProfileDesigner.DAL/ProfileTypeDefinitionDAL.cs

```diff
@@ -223,7 +223,7 @@ private static bool MatchIdentity(int? entityId, string entityOpcNodeId, string
             }
             else
             {
-                return modelOpcNodeId == entityOpcNodeId && modelNamespace == entityNamespace;
+                return (modelOpcNodeId != null || entityOpcNodeId != null) && modelOpcNodeId == entityOpcNodeId && modelNamespace == entityNamespace;
```

Comment:
Discussion on the browsename change in attributeservice.js: I think this change makes the absence of a browsename work: before we would latch on the first empty/null browsename.